runway/module/__init__.py

```diff
@@ -121,3 +138,53 @@ def destroy(self):
         """Implement dummy method (set in consuming classes)."""
         raise NotImplementedError('You must implement the destroy() method '
                                   'yourself!')
+
+
+class RunwayModuleFolder(object):
+    """Functions to manage filesystem access in a module folder."""
+
+    def __init__(self, path):
+        """Initialize base class."""
+        self._path = path
+
+    def fullpath(self, name):
+        """Return the absolute path to the given file."""
+        return os.path.join(self._path, name)
+
+    def isfile(self, name):
+        """Determine if the given file exist relative to the module."""
+        return os.path.isfile(self.fullpath(name))
+
+    def isdir(self, name):
+        """Determine if the given folder exist relative to the module."""
+        return os.path.isdir(self.fullpath(name))
+
+    def locate_file(self, names):
+        """Given a list of files, find one that exists (if any) in the root folder."""
+        for name in names:
+            if self.isfile(name):
+                return name
+        # IDEA: it might be better to find *all* of the existing files,
+        #  and log a warning if more than one is found?
+        return None
+
+    def locate_env_file(self, names):
```

Comment:
Some modules use 'env' some don't.... may as well let them all do it.   And we could add other folders here in the future.